articles/api/authentication/_logout.md

```diff
@@ -24,50 +32,62 @@ curl --request GET \
+Use this endpoint to logout a user. If you want to navigate the user to a specific URL after the logout, set that URL at the `post_logout_redirect_uri` parameter. The URL should be included in any the appropriate `Allowed Logout URLs` list:
+
+- If the `client_id` parameter is included, the `post_logout_redirect_uri` URL must be listed in the `Allowed Logout URLs` set at the application level (see [Setting Allowed Logout URLs at the App Level](https://auth0.com/docs/logout#set-the-allowed-logout-urls-at-the-application-level)).
+- If the `client_id` parameter is NOT included, the `post_logout_redirect_uri` URL must be listed in the `Allowed Logout URLs` set at the tenant level (see [Setting Allowed Logout URLs at the Tenant Level](https://auth0.com/docs/logout#set-the-allowed-logout-urls-at-the-tenant-level)).
+- If the `client_id` parameter is included and the `post_logout_redirect_uri` URL is NOT set, the server returns the user to the first `Allowed Logout URLs` set in the Dashboard (see [Setting Allowed Logout URLs at the Tenant Level](https://auth0.com/docs/logout#set-the-allowed-logout-urls-at-the-tenant-level)).
 
 
 ### Request Parameters
 
-| Parameter        | Description |
-|:-----------------|:------------|
-| `returnTo `      | URL to redirect the user after the logout. |
-| `client_id`      | The `client_id` of your application. |
-| `federated`      | Add this query string parameter to the logout URL, to log the user out of their identity provider, as well: `https://${account.namespace}/v2/logout?federated`. |
+| Parameter                             | Description                                                                                                                                                     |
+| :------------------------------------ | :-------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id_token_hint` (Recommended)         | Previously issued ID Token for the user. This is used to indicate which user to log out                                                                         |
+| `logout_hint` (Optional)              | Optional `sub` (user ID) or `sid` (session ID) value to indicate which user to log out.                                                                         |
+| `post_logout_redirect_uri` (Optional) | URL to redirect the user after the logout.                                                                                                                      |
+| `client_id` (Optional)                | The `client_id` of your application.                                                                                                                            |
+| `federated` (Optional)                | Add this query string parameter to the logout URL, to log the user out of their identity provider, as well: `https://YOUR_DOMAIN/oidc/logout?federated`.        |
```

Comment:
Add this query string parameter to log the user out of their identity provider: `https://YOUR_DOMAIN/oidc/logout?federated`.